app/assets/stylesheets/application.scss

```diff
@@ -63,6 +63,13 @@ table {
   border: solid thin black
 }
 
+dd.head, dt.head {
```

Comment:
What does 'head' stand for in this context?

Reply:
Well, other than just being a css class... it is for the table/list at the top of the page...
So, the "table" (which isn't a table) that shows event, session, etc. 
Just a not-great name, @panpear